src/test/java/com/microsoft/azure/proton/transport/proxy/impl/BasicProxyChallengeProcessorImplTest.java

```diff
@@ -41,15 +41,17 @@ public void setup() {
 
             @Override
             public void connectFailed(URI uri, SocketAddress sa, IOException ioe) {
-                System.out.format("PROXY CONNECTION FAILED: URI = %s, Socket Address = %s, IO Exception = %s\n", uri.toString(), sa.toString(), ioe.toString());
+                System.out.format("PROXY CONNECTION FAILED: URI = %s, Socket Address = %s, IO Exception = %s\n", uri.toString(), sa.toString(),
```

Comment:
nit: it's not part of your PR, but could you update the format string to use `%n` rather than `\n`.

While it's fresh in my mind. Thanks! :D

Reply:
Done